src/Tizen.NUI/src/public/Window/GLWindow.cs

```diff
@@ -443,13 +443,13 @@ public void RenderOnce()
         HandleRef TerminateHandlerRef;
 
         /// <summary>
-        /// Registers a GL callback function for application.
+        /// Registers a GL callback functions for application.
```

Comment:
If you want to change the word to the plural, how about removing 'a' character? 
```suggestion
        /// Registers GL callback functions for application.
```